src/pages/versions.js

```diff
@@ -28,20 +28,20 @@ const Versions = ({location}: Props) => (
           <TitleAndMetaTags title="React - Versions" />
           <div css={sharedStyles.markdown}>
             <p>
-              A complete release history for React is available{' '}
+              Полная история версий React доступна{' '}
               <a
                 href="https://github.com/facebook/react/releases"
                 target="_blank"
                 rel="noopener">
-                on GitHub
+                на GitHub
               </a>
               .<br />
-              Documentation for recent releases can also be found below.
+              Документация к последним версиям может быть также найдена ниже.
             </p>
             <p>
-              See our FAQ for information about{' '}
+              Секция FAQ содержит информацию о {' '}
```

Comment:
Получается, что лишний пробел по сравнению с оригиналом
```suggestion
              Секция FAQ содержит информацию о{' '}
```